hypnos/src/clock.c

```diff
@@ -33,17 +36,17 @@ static struct tm ti = {
 /* ********** ********** ********** ********** ********** ********** */
 
 /* ********** ********** FUNCTIONS ********** ********** */
-void clock_str_to_local_time(const char *str, struct tm *ti)
+void clock_str_to_local_time(const char *str, struct tm *t)
```

Comment:
Hard to understand what "t" is. Consider using something like "time" instead.

Reply:
I can't call it `time` because then GCC says: `/usr/include/newlib/time.h:59:11: note: previous declaration of 'time' was here`.